weather/test/handle_weather_data_tests.py

```diff
@@ -0,0 +1,20 @@
+from weather.core.handle_weather_data import write_content_to_json, get_webpage_content_in_json_format
+import unittest
+import os
+
+class MyTestCase(unittest.TestCase):
+    def setUp(self):
+        self.mock_dict = {"FeelsLikeC": "5",
+                          "FeelsLikeF": "41",
+                          "cloudcover": "15",
+                          "humidity": "88",
+                          }
+
+    def test_get_webpage_content_in_json_format(self):
+        self.assertTrue(type(get_webpage_content_in_json_format("http://wttr.in/")) is dict)
```

Comment:
Ennek a tesztnek nicns értelme. Azt kellene inkább nézni, hogy úgy reagál e a funkciód/modulod/applikációd a különböző előfordulható esetekre ahogy te is gondolod. De itt talán a legnagyobb probléma az infrastruktúrára dependálás. Kezdjük azzal, hogy azt kiszeded. Utánna jöhetnek ezek a kérdések:
- Mi van ha a link jó és megpróbálsz requestelni?
- Mi van ha Rossz a link? itt lehet olyan, hogy szerepel benne olyan karakter aminek nem lenne szabad, nem teljes a link... stb
- Mi van ha nem 200-at ad vissza az oldal? 
- Az amit válaszol ezere az appod az user frendly?

Reply:
Mi van ha jó a link siekres a request de üres a válasz?

Reply:
Illetve légy kreativ :D